tests/unit/constraints/test_base.py

```diff
@@ -159,6 +159,21 @@ def test_import_object_function():
 
 class TestConstraint():
 
+    def test__validate_inputs(self):
+        """Test the ``_validate_inputs`` method.
+
+        The method should only raise errors if the input paramaters are invalid.
+
+        Raise:
+            - ``MultipleConstraintsErrors`` if errors were found.
+        """
+        # Run
+        Constraint._validate_inputs(args='value', kwargs='value')
+
+        # Run / Assert
+        with pytest.raises(MultipleConstraintsErrors):
```

Comment:
minor: maybe we should match the error like we've been doing to avoid more changes when we add linting